libcxx/test/std/containers/views/mdspan/layout_stride/index_operator.pass.cpp

```diff
@@ -53,9 +53,9 @@ constexpr void iterate_stride(M m, const std::array<int, M::extents_type::rank()
   constexpr int r = static_cast<int>(M::extents_type::rank()) - 1 - static_cast<int>(sizeof...(Args));
   if constexpr (-1 == r) {
     ASSERT_NOEXCEPT(m(args...));
-    size_t expected_val = [&]<size_t... Pos>(std::index_sequence<Pos...>) {
+    size_t expected_val = static_cast<size_t>([&]<size_t... Pos>(std::index_sequence<Pos...>) {
```

Comment:
While you're at it can you add `std::` to `size_t` in this area too?

Reply:
Done, pushed a commit to qualify all occurrences in this file.